readme.md

```diff
@@ -127,6 +127,23 @@ Custom promise module to use instead of the native one.
 - Pify has useful options like the ability to handle multiple arguments (`multiArgs`).
 - Pify does not have [magic behavior](https://nodejs.org/api/util.html#util_custom_promisified_functions) for certain Node.js methods and instead focuses on predictability.
 
+#### Why can't a promisified class function access it's own context?
+
+The context of free functions can't be detected. As a workaround, promisify the whole class and extract the function from the result.
+
+```js
+const pify = require('pify');
+const SomeClass = require('./some-class');
+
+const someClass = new SomeClass();
+
+// `someFunction` can't access it's class context
+const someFunction = pify(someClass.someFunction);
+
+// `someFunction` is extracted from the entire promisified class, allowing it to keep it's context
+const someFunction = pify(someClass).someFunction;
+```
```

Comment:
Maybe we should also mention that you can use `.bind`. `const someFunction = pify(someClass.someFunction.bind(someClass));`

Reply:
Fixed!